internal/provider/telemetry_resource.go

```diff
@@ -140,14 +145,23 @@ func (r *TelemetryResource) Create(ctx context.Context, req resource.CreateReque
 		return
 	}
 
+	data.ModuleSource = basetypes.NewStringNull()
```

Comment:
As above, can we fallback to checking for a supplied source and version if the module_path is not supplied.

Reply:
Resolved in 6582745